iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/HvdcLineImpl.java

```diff
@@ -231,6 +235,51 @@ public void remove() {
         network.getListeners().notifyAfterRemoval(id);
     }
 
+    @Override
+    public boolean connectConverterStations() {
+        return connectConverterStations(SwitchPredicates.IS_NONFICTIONAL_BREAKER, null);
+    }
+
+    @Override
+    public boolean connectConverterStations(Predicate<Switch> isTypeSwitchToOperate) {
+        return connectConverterStations(isTypeSwitchToOperate, null);
+    }
+
+    @Override
+    public boolean connectConverterStations(Predicate<Switch> isTypeSwitchToOperate, TwoSides side) {
```

Comment:
Shouldn't we rename this to `connectConverterStation`? It's strange to give a side and have the plural form, and also to interpret the null value as both sides.

Reply:
For the null value, it's already like this for the connectables so I kept the same logic. Same for the plural form : it's `getTerminals(ThreeSides side)` even when we give a ThreeSide so I tried to keep it coherent